src/components/About Us/about.scss

```diff
@@ -0,0 +1,54 @@
+
+
+.ABOUT {
+  font-size: 32px;
+  text-decoration: bold;
+	line-height: 33px;
+	color: #000;
+
+.About {
```

Comment:
This is bad practice to have classNames named the same but with different cases. Try something like this

```css
.about-container {
	...
	.about {
		...
	}
}
```